tgui/packages/tgui/interfaces/Who.jsx

```diff
@@ -44,7 +48,7 @@ export const Who = (props, context) => {
                     fluid
                     onEnter={(e, value) =>
```

Comment:
Nothing happens if you enter a valid ckey and then press enter (as in no player panel opens).

Additionally, if you enter an invalid ckey you get:
![image](https://github.com/user-attachments/assets/3e7c4709-d262-4b4a-b17f-57481eff9eab)
![image](https://github.com/user-attachments/assets/17d4420c-bf51-473a-bc18-278d759a5584)